src/routes/balance/components/index.tsx

```diff
@@ -106,6 +106,7 @@ class BalanceLayout extends React.Component<Props, State> {
     const info: Order = { xs: 2 };
     const grow: Order = { xs: 3 };
     const hasTokens = tokens && tokens.length;
+    const sortedTokens = [...tokens].sort((a, b) => a.tokenTicker.localeCompare(b.tokenTicker));
```

Comment:
This sorting should be done in the selector, for having memoized them. Here:
https://github.com/iov-one/wallet-demo/blob/master/src/routes/balance/container/selector.ts#L14